lib/stats_reporter.rb

```diff
@@ -0,0 +1,117 @@
+# generates text for a report on storage root size and object status
+class StatsReporter
+
+  require 'text-table'
+  require 'faraday'
+  require 'open3'
+
+  REPOSITORY = 'sdr'.freeze
+  INGEST_WF = 'preservationIngestWF'.freeze
+  INGEST_WF_STEPS = [
+    'start-ingest',
+    'transfer-object',
+    'validate-bag',
+    'verify-apo',
+    'update-moab',
+    'update-catalog',
+    'complete-ingest'
+  ].freeze
```

Comment:
it's not such a terrible idea to parse these out from config/workflows/sdr/preservationIngestWF.xml, but I'm okay with the hardcoding at this time.